app/Http/Controllers/Administration/UserController.php

```diff
@@ -130,4 +130,48 @@ public function getEmail(): array
 			'email' => AccessControl::user()->email,
 		];
 	}
+
+	/**
+	 * Returns the currently authenticated user.
+	 *
+	 * @return User
+	 */
+	public function getCurrent(): User
+	{
+		return AccessControl::user();
+	}
+
+	/**
+	 * Reset the token of the currently authenticated user.
+	 *
+	 * @return array
+	 *
+	 * @throws \Exception
+	 */
+	public function resetToken(): array
+	{
+		AccessControl::user()->token = strtr(base64_encode(random_bytes(16)), '+/', '-_');
+		AccessControl::user()->save();
+
+		return [
+			'token' => AccessControl::user()->token,
+		];
+	}
+
+	/**
+	 * Disable the token of the currently authenticated user.
+	 *
+	 * @return array
+	 *
+	 * @throws \Exception
+	 */
+	public function disableToken(): array
+	{
+		AccessControl::user()->token = '';
+		AccessControl::user()->save();
+
+		return [
+			'token' => '',
+		];
+	}
```

Comment:
Why do we need to return anything at all? Also I would prefer `unsetToken` because it matches `resetToken`.

```suggestion
	public function unsetToken(): void
	{
		$user = AccessControl::user();
		$user->token = null;
		$user->save();
	}
```

This will return `204 - No Content` in case of success and a proper error code in any other case.

If you wonder why I would like to use `null` instead of the empty string `'' ` see my other comments about unique constraints and DB query efficiency.